server/modules/surveyRdb/api/surveyRdbApi.js

```diff
@@ -34,6 +37,32 @@ export const init = (app) => {
     }
   })
 
+  app.post('/surveyRdb/:surveyId/:nodeDefUuidTable/chart', requireRecordListViewPermission, async (req, res, next) => {
```

Comment:
maybe we should extract this to other service to encapsulate Vega

Reply:
do you mean the apis? yes, it is something separate from rdb